lib/src/providers/VernaProvider/schemaFormActions.tsx

```diff
@@ -0,0 +1,125 @@
+import { ObjectFieldTemplateProps } from '@rjsf/core';
```

Comment:
I'll rename this file `useSchemaReducer` and move it in a dedicated folder.
e.g: `/src/reducers/useSchemaReducer`